### PR TITLE
Acknowledge efforts to port the pills to its current format

### PR DESCRIPTION
**Pull request by DanielSidhion**
Important to acknowledge all the work that went into the port!

cc @noamraph @jtojnar let me know if you don't want to be credited directly

### CODE REVIEW

**Comment by jtojnar** [comment]
DocBook should be capitalized like this.

File: pills/00-preface.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
 # Preface
 
-This is a ported version of the **Nix Pills**, a series of blog posts written by **Luca Bruno** (aka Lethalman) and originally published in 2014 and 2015. It provides a tutorial introduction into the Nix package manager and Nixpkgs package collection, in the form of short chapters called 'pills'.
+This is a ported version of the **Nix Pills**, a series of blog posts written by **Luca Bruno** (aka Lethalman) and originally published in 2014 and 2015.
+It provides a tutorial introduction into the Nix package manager and Nixpkgs package collection, in the form of short chapters called 'pills'.
 
-Since the Nix Pills are considered a classic introduction to Nix, an effort to port them to the current format was led by Graham Christensen (aka grahamc / gchristensen) and other contributors in 2017.
+Since the Nix Pills are considered a classic introduction to Nix, an effort to port them to a DocBook format was led by Graham Christensen (aka grahamc / gchristensen) and other contributors in 2017.
+In 2024, Noam Yorav-Raphael (aka noamraph) and Jan Tojnar (aka jtojnar) ported them to the current mdBook and EPUB formats.
```
Comment (noamraph) [inline pills/00-preface.md:7]: Perhaps replace "the current mdBook and EPUB formats" with just "the current mdBook format"? The source format is mdBook, EPUB is an output format, just like HTML.

**Comment by noamraph** [comment]
Thank you @DanielSidhion ! I'm honored to be credited, and I'm happy that @jtojnar is credited as well for his hard work.

File: pills/00-preface.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
 # Preface
 
-This is a ported version of the **Nix Pills**, a series of blog posts written by **Luca Bruno** (aka Lethalman) and originally published in 2014 and 2015. It provides a tutorial introduction into the Nix package manager and Nixpkgs package collection, in the form of short chapters called 'pills'.
+This is a ported version of the **Nix Pills**, a series of blog posts written by **Luca Bruno** (aka Lethalman) and originally published in 2014 and 2015.
+It provides a tutorial introduction into the Nix package manager and Nixpkgs package collection, in the form of short chapters called 'pills'.
 
-Since the Nix Pills are considered a classic introduction to Nix, an effort to port them to the current format was led by Graham Christensen (aka grahamc / gchristensen) and other contributors in 2017.
+Since the Nix Pills are considered a classic introduction to Nix, an effort to port them to a DocBook format was led by Graham Christensen (aka grahamc / gchristensen) and other contributors in 2017.
```
Comment (drupol) [inline pills/00-preface.md:6]: Why 2 pseudos for Graham ?

Comment (DanielSidhion) [reply pills/00-preface.md:6]: I don't know, I'm just keeping the information that was there before.

Comment (drupol) [reply pills/00-preface.md:6]: @grahamc Should we keep the 2 pseudonames or one is enough here?

Comment (grahamc) [reply pills/00-preface.md:6]: grahamc is great, thanks! :)

**Pull request closed without merging**